samples/Face/Microsoft.Azure.CognitiveServices.Samples.Face.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFrameworks>netcoreapp2.1;netstandard1.4</TargetFrameworks>
```

Comment:
Add `netstandard1.4` to ensure compatibility, this is the the target the tester uses.